services/movie.py

```diff
@@ -0,0 +1,23 @@
+from db.models import Movie, Genre, Actor
+
+
+def get_movies(genres_ids=None, actors_ids=None):
+    queryset = Movie.objects.all()
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+    return queryset.distinct()
+
+
+def get_movie_by_id(movie_id):
+    return Movie.objects.get(id=movie_id)
```

Comment:
Consider handling the case where the `movie_id` does not exist. Using `get` will raise a `Movie.DoesNotExist` exception if the movie is not found. You might want to handle this exception or use `Movie.objects.filter(id=movie_id).first()` to return `None` if the movie is not found.